internal/gorson/bash/bash.go

```diff
@@ -21,9 +21,8 @@ func ParamsToShell(parameters map[string]string) string {
 	keys := getKeys(parameters)
 	sort.Strings(keys)
 	for _, key := range keys {
-		// TODO do we need to do extra escaping here?
 		v := parameters[key]
-		lines = append(lines, "export "+key+"=\""+v+"\"")
+		lines = append(lines, "export "+key+"='"+v+"'")
```

Comment:
I wonder if we should do something like this...

```
v := parameters[key]
line := fmt.Sprintf("export %s='%s'", key, v)
lines = append(lines, line)
```

Reply:
good idea, done in 0bee6da086b0